Makefile

```diff
@@ -98,12 +98,12 @@ build: checkenv build-adapter build-controller
 .PHONY: build-controller
 build-controller: generate-api pkg/scalers/liiklus/LiiklusService.pb.go
 	$(GO_BUILD_VARS) operator-sdk build $(IMAGE_CONTROLLER) \
-		--go-build-args "-ldflags -X=main.GitCommit=$(GIT_COMMIT) -ldflags -X=github.com/kedacore/keda/version.Version=$(VERSION) -o build/_output/bin/keda"
+		--go-build-args "-ldflags -X=github.com/kedacore/keda/version.GitCommit=$(GIT_COMMIT) -ldflags -X=github.com/kedacore/keda/version.Version=$(VERSION) -o build/_output/bin/keda"
```

Comment:
I think this doesn't actually work, right? The old one didn't, `GitCommit` was always empty. I looked up the issue and had a change for it, but I never pushed it. `operator-sdk` doesn't support multiple AFAIK https://github.com/operator-framework/operator-sdk/issues/1867. You can just remove the second `-ldflags` arg

Reply:
Haven't noticed that, thanks!

Reply:
Fixed